internal/state/execution.go

```diff
@@ -87,9 +86,8 @@ func (blockExec *BlockExecutor) CreateProposalBlock(
 	ctx context.Context,
 	height int64,
 	state State,
-	commit *types.Commit,
+	lastExtCommit *types.ExtendedCommit,
```

Comment:
Could be `lastCommit` probably.

Reply:
We prefer to distinguish between "commits" and "extended commits" all over the code.
`lastEC` maybe?